backends/ebpf/targets/ebpfenv.py

```diff
@@ -29,66 +29,77 @@
 
 class Bridge:
 
-    def __init__(self, namespace):
-        self.ns_name = namespace # identifier of the namespace
-        self.br_name = "core"    # name of the central bridge
-        self.br_ports = []       # list of the veth pair bridge ports
-        self.edge_ports = []     # list of the veth pair edge ports
-
-    def ns_init(self):
+    def __init__(self, namespace: str):
+        # Identifier of the namespace.
```

Comment:
we have too many bugs to fix to spend the time to reformat the way comments look.
and to review that code. that's a lot of wasted energy.